doc/advanced/content/executor_design.rst

```diff
@@ -58,9 +58,9 @@ the maintenance complexity of the cross product of a set of parallel algorithms
 Execution created through different facilities has different synchronization properties.
 For example, an OpenMP parallel for loop is synchronous because the spawning thread blocks until the loop
 is complete due to an implicit barrier at the end of the parallel region by default. In contrast, the execution
-of GPU kernels is typically asynchronous; kernel launches return immediately and the launching thread
-continues its execution without waiting for the kernel’s execution to complete. Work submitted to a thread
-pool may or may not block the submitting thread. Correct code must account for these synchronization
+of GPU kernels is typically asynchronous; kernel launches return immediately, and the launching thread
+continues its execution without waiting for the kernel's execution to complete. Work submitted to a thread
+pool may or may not block the submitting thread. The correct code must account for these synchronization
```

Comment:
questionable change

Reply:
`A` correct code?

Reply:
I would stick to the original in this case. Simply

Correct code must account...